src/h5web/vis-packs/core/heatmap/interpolators.ts

```diff
@@ -38,12 +39,17 @@ import {
   interpolateTurbo,
 } from 'd3-scale-chromatic';
 
+function interpolateHsl(t: number): string {
+  return hsl(t * 360, 1, 0.5).formatRgb();
+}
+
 export const MOST_USED = {
```

Comment:
Could we rename this in `COMMON` ?

Reply:
Or should we change the label of the group in the drop-down?

Reply:
Ah no, we talked about it, I remember. It sounded like it was the most used by the user. I'll rename the constant 👍